### PR TITLE
Better sights (Plus some attachment stats)

### PR DESCRIPTION
**Pull request by VolatileIvory**
Same thing as UR, just made sights comfortably distanced from the camera.

### CODE REVIEW

**Comment by VolatileIvory** [comment]
Gave the M1014 grip stock stats just because, and made it and the sporting stock have slightly fancier titles. Because why not.

File: lua/weapons/arccw_ud_m1014.lua [hunk 1]
```diff
@@ -347,14 +347,14 @@ SWEP.Animations = {
...
+        ShellEjectAt = 1.1,
         LHIK = true,
         SoundTable = {
             {s = rottle, t = 0},
-            {s = path2 .. "grab.ogg", t = 0.1},
-            {s = path .. "chback.ogg", t = 0.7},
-            {s = path1 .. "eject.ogg", t = 0.8, v = 0.4},
-            {s = path .. "chamber.ogg", t = 1.0},
+            {s = path2 .. "grab.ogg", t = .4},
+            {s = path2 .. "usas_chback.ogg", t = 0.8},
```
Comment (TheOnly8Z) [inline lua/weapons/arccw_ud_m1014.lua:355]: Looks like a sound from the deprecated USAS - Is it appropriate here?

File: lua/arccw/shared/attachments/ud_m16_stock_moe.lua [hunk 2]
```diff
@@ -1,12 +1,8 @@
-att.PrintName = "AMCAR MOE Stock"
+att.PrintName = "Magpul MOE Stock"
```
Comment (TheOnly8Z) [inline lua/arccw/shared/attachments/ud_m16_stock_moe.lua:1]: Real brand should only show up with true names on. Please keep as AMCAR for consistency across M16 attachments and set true name to MAGPUL (upper case)

File: lua/arccw/shared/attachments/ud_m1014_stock_sport.lua [hunk 3]
```diff
@@ -1,6 +1,7 @@
-att.PrintName = "Sporting Stock"
+att.PrintName = "M4 Semi-Grip Fixed Stock"
```
Comment (TheOnly8Z) [inline lua/arccw/shared/attachments/ud_m1014_stock_sport.lua:1]: Use true names only when truename convar is set.

**Review by TheOnly8Z — CHANGES_REQUESTED** [review]

**Comment by Fesiug** [comment]
If you are still developing this, join the Discord and talk with us there

**Pull request closed without merging**